libkirk/main.py

```diff
@@ -405,131 +405,124 @@ def run(cmd_args: list = None) -> None:
     parser = argparse.ArgumentParser(
         description='Kirk - All-in-one Linux Testing Framework')
 
-    # generic arguments
-    parser.add_argument(
+    generic_opts = parser.add_argument_group('General options')
+    generic_opts.add_argument(
         "--version",
         "-V",
         action="version",
         version=f"%(prog)s, {__version__}")
-
-    # user interface arguments
-    parser.add_argument(
+    generic_opts.add_argument(
         "--verbose",
         "-v",
         action="store_true",
         help="Verbose mode")
-    parser.add_argument(
+    generic_opts.add_argument(
         "--no-colors",
         "-n",
         action="store_true",
         help="If defined, no colors are shown")
-
-    # generic directories arguments
-    parser.add_argument(
+    generic_opts.add_argument(
         "--tmp-dir",
         "-d",
         type=str,
         default="/tmp",
         help="Temporary directory")
-    parser.add_argument(
+    generic_opts.add_argument(
         "--restore",
-        "-R",
+        "-r",
         type=str,
         help="Restore a specific session")
+    generic_opts.add_argument(
+        "--json-report",
+        "-o",
```

Comment:
I'm ok with this, but `-j` was also understandable.

Reply:
I kept `-o` because in this case we don't support any output format but JSON.